hack/lint.sh

```diff
@@ -14,14 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-HACK_DIR=$(dirname "${BASH_SOURCE[0]}")
-
 # Printing out cache status
 golangci-lint cache status
 
 # Enable GL_DEBUG line below for debug messages for golangci-lint
 # export GL_DEBUG=loader,gocritic,env
-CMD="golangci-lint run -c $HACK_DIR/../golangci.yaml"
+CMD="golangci-lint run $@"
```

Comment:
Would be great to note that --fix can be passed in here, and that it may not fix every issue.

Reply:
I think I already added this in here https://github.com/vmware-tanzu/velero/pull/7194/files#diff-3148449172665e1fe59b7d02c5f38a0053ad8e689cc457569092bb395f2312a0